CONTRIBUTING.md

```diff
@@ -257,17 +165,40 @@ into the source repository.
   area of the PR. You can also advertise the PR on devel@riot-os.org mailing
   list and ask for a review there.
 
+* Try to answer reviews as quickly as possible to speed up the review process
+  and avoid stalled PRs.
+
 You can find more information about RIOT development procedure on this
 [wiki page][development-procedures].
```

Comment:
this link was and still is broken.

Reply:
If you have the right link I'll open a separate PR fixing it!